CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
Original referenced version from [contributor-covenant.org](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html) has in addition "... level of experience, **education, socio-economic status**, nationality..." GitHub template might be outdated, should we add them?

Reply:
Yep, will do a change.